### PR TITLE
Marketing page: Add 'become a host' page

### PR DESCRIPTION
**Pull request by flickz**
Path - `/become-a-host`

Figman Ref - https://www.figma.com/file/ZQBMWhnGGtRWeIZknFW1eP/%5BOC.com%5D-Production-Ready-%E2%9C%85?node-id=8274%3A1003

### CODE REVIEW

File: components/Grid.js [hunk 1]
```diff
@@ -5,9 +5,9 @@
 
 import propTypes from '@styled-system/prop-types';
 import styled from 'styled-components';
-import { color, compose, flexbox, grid, layout, space, typography } from 'styled-system';
+import { background, color, compose, flexbox, grid, layout, space, typography } from 'styled-system';
 
-const boxProps = compose(space, color, layout, typography, flexbox, grid);
+const boxProps = compose(space, color, layout, typography, flexbox, grid, background);
```
Comment (Betree) [inline components/Grid.js:10]: We should avoid extending this component, we should rather use `Container` for things that need more customization

**Review by Betree — COMMENTED** [review]
The code looks good. Only found one small responsiveness issue, I suppose images should be centered here right?

image

File: components/Grid.js (hunk 1 above)
Comment (flickz) [reply components/Grid.js]: Gotcha!

**Comment by flickz** [comment]
@Betree The responsive issue should be fixed now.

File: server/pages.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ const routes = require('next-routes');
 const pages = routes()
   .add('home', '/', 'index')
   .add('become-a-sponsor', '/become-a-sponsor', 'become-a-sponsor')
+  .add('become-a-host', '/(become-a-sponsor|become-a-fiscal-host)', 'become-a-host')
```
Comment (Betree) [inline server/pages.js:12]: I don't get why `become-a-sponsor` is here?

Comment (flickz) [reply server/pages.js:12]: It was a mistake, thanks.

**Comment by flickz** [comment]
@Betree All Good!

**Comment by Betree** [comment]
Looks great, merging now

**Pull request merged**